web/README.md

```diff
@@ -1,3 +1,324 @@
+| ready | Animation data is loaded and player is ready |
+| play | Animation starts playing |
+| pause | Animation is paused |
+| stop | Animation is stopped |
+| freeze | Animation is paused due to player being invisible |
+| loop | An animation loop is completed |
+| complete | Animation is complete (all loops completed) |
+| frame | A new frame is entered |
+
+
+## Methods
+
+**Method** : `load(src: string | object)`
+
+**Purpose** : Load and play LottieFiles animation
+
+**Parameters**
+| Name | Type | Description
+| --- | --- | --- |
+| src | `string` or `object` | URL, or a JSON string or object representing a LottieFiles format to play.
+
+**Return Type** : `Promise<void>`
+
+---
+
+**Method** : `play()`
+
+**Purpose** : Play loaded animation
+
+**Return Type** : `void`
+
+---
+
+**Method** : `pause()`
+
+**Purpose** : Pause playing animation
+
+**Return Type** : `void`
+
+---
+
+**Method** : `stop()`
+
+**Purpose** : Stop current animation
+
+**Return Type** : `void`
+
+---
+
+**Method** : `seek(frame: number)`
+
+**Purpose** : Move to given frame
+
+**Parameters**
+| Name | Type | Description
+| --- | --- | --- |
+| src | `number` | The frame number to move, shouldn't be less than 0 and greater than `totalFrame`
+
+**Return Type** : `void`
+
+> [!TIP]
+> You can easily check total frame of animation, use `player.totalFrame`
+
+---
+
+**Method** : `destory()`
+
+**Purpose** : Destroy animation and lottie-player element
+
+**Return Type** : `void`
+
+---
+
+**Method** : `setLooping(value: boolean)`
+
+**Purpose** : Enable animation loop
+
+**Parameters**
+| Name | Type | Description
+| --- | --- | --- |
+| value | `boolean` | `true` enables looping, while `false` disables looping.
+
+**Return Type** : `void`
+
+---
+
+**Method** : `setDirection(value: number)`
+
+**Purpose** : Set animation direction
+
+
+**Parameters**
+| Name | Type | Description
+| --- | --- | --- |
+| value | `number` | Direction values. `1` to play the animation forward, `-1` to play it backward.
+
+**Return Type** : `void`
+
+---
+
+**Method** : `setSpeed(value: number)`
+
+**Purpose** : Set speed of animation
+
+
+**Parameters**
+| Name | Type | Description
+| --- | --- | --- |
+| value | `number` | Playback speed. The value must be any positive integer.
+
+**Return Type** : `void`
+
+---
+
+**Method** : `setBgColor(value: string)`
+
+**Purpose** : Set a backgroud color
+
+**Parameters**
+| Name | Type | Description
+| --- | --- | --- |
+| value | `string` | Color values. Hex(#fff) or string('red') to set background color
+
+**Return Type** : `void`
+
+---
+
+**Method** : `save(target: ExportableType)`
+
+**Purpose** : Export current animation to other file type
+
+**Parameters**
+| Name | Type | Description
+| --- | --- | --- |
+| value | `ExportableType` | File type to export
+
+**Return Type** : `void`
+
+
+---
+
+**Method** : `getVersion()`
+
+**Purpose** : Return current ThorVG version
+
+**Return Type** : `LibraryVersion`
+
+
+
+
+
+# Build
```

Comment:
Do users need to know about wasm build? I expect the prenbuilt wasm is in the distributed package.

Reply:
This information is for the contributor and developer wants to build themselves, so the users who will use only library will not need build step, plus prebuilt wasm will be definitely distributed.
(_In other words, we don't need to build when using this library_)

Normally, npm package and it's document contain explanation of build steps, and this library strongly depends on ThorVG, I think we should provide users with option for the standalone build.

Reply:
If you think this README.md is necessary for npm package, we don't release the package for ThorVG contributors/developers. We can prepare the wiki for developers; they can visit the ThorVG community to collaborate on it. Let's focus on the guide for those likely to use the distributed package.

Reply:
That's better! I will move this info under the Wiki. Please get me to wiki permission later, seems I'm not able to create page

Reply:
That's better! I will move this info under the Wiki.

Please get me wiki permission later, seems I'm not able to create page.